libtiff/libtiff_ctypes.py

```diff
@@ -966,6 +995,58 @@ def WriteEncodedStrip(self, strip, buf, size):
         assert r.value == size, repr((r.value, size))
     writeencodedstrip = WriteEncodedStrip
 
+    @debug
+    def ReadTile(self, buf, x, y, z, sample):
+        """ Read and decode a tile of data from an open TIFF file
+
+        Parameters
+        ----------
+        buf: array
+            contiguous array of bytes
+        x: int
+        y: int
+        z: int
+            It is used if the image is deeper than 1 slice (ImageDepth>1)
+        sample: integer
+            It is used only if data are organized
+            in separate planes (PlanarConfiguration=2)
+
+        Returns
+        -------
+        int
+            -1 if it detects an error;
+            otherwise the number of bytes in the decoded tile is returned.
+        """
+        return libtiff.TIFFReadTile(self, buf, x, y, z, sample)
+    readtile = ReadTile
```

Comment:
Also no need to create a "readtile" (lowercase) method, nor a "writetile" method.